examples/with-react-intl/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "react-intl-example-next",
+  "version": "1.0.0",
+  "description": "React Intl Next.js Example App",
+  "author": "Eric Ferraiuolo <edf@ericf.me>",
+  "license": "BSD-3-Clause",
```

Comment:
All examples have
- `name` the same as the directory in this case `with-react-intl`
- No `description`
- Empty author (`"author": ""`)
- an `ISC` license https://github.com/zeit/next.js/blob/master/examples/basic-css/package.json#L15